src/OpenLoco/Vehicles/Vehicle1.cpp

```diff
@@ -0,0 +1,16 @@
+#include "../Interop/Interop.hpp"
+#include "Vehicle.h"
+
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::Vehicles
+{
+    // 0x004A9788
+    bool Vehicle1::update()
+    {
+        registers regs;
+        regs.esi = X86Pointer(this);
+
+        return !(call(0x004A9788, regs) & X86_FLAG_CARRY);
```

Comment:
the old code had these compare `!= 0;`, was there any reason to change it to just a 'not? (I know they're functionally equivalent)

Reply:
I personally prefer just the !. But its mainly just copy paste reasons.

Reply:
no worries